test/functional/feature_csv_activation.py

```diff
@@ -2,484 +2,25 @@
 # Copyright (c) 2015-2018 The Bitcoin Core developers
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
-"""Test activation of the first version bits soft fork.
+"""Test activation of CSV at genesis for Elements."""
 
-This soft fork will activate the following BIPS:
```

Comment:
Is it smart to remove all this instead of disable somehow? I'm not sure, but I think this might make upstream updates harder.

Reply:
I could remove the test from test_runner, then add a specific check in a custom test?

Reply:
That'd work as well.

Reply:
rather than disable the test, I just reverted all the changes and had the functional suite default to bitcoin-regtest argument